modules/webgl/src/classes/framebuffer.js

```diff
@@ -23,12 +23,24 @@ export default class Framebuffer extends Resource {
     } = {}
   ) {
     let supported = true;
-    supported =
-      colorBufferFloat &&
-      gl.getExtension(isWebGL2(gl) ? 'EXT_color_buffer_float' : 'WEBGL.color_buffer_float');
-    supported =
-      colorBufferHalfFloat &&
-      gl.getExtension(isWebGL2(gl) ? 'EXT_color_buffer_float' : 'EXT_color_buffer_half_float');
+
+    if (colorBufferFloat) {
+      supported = Boolean(
+        gl.getExtension('EXT_color_buffer_float') ||
+          gl.getExtension('WEBGL_color_buffer_float') ||
+          gl.getExtension('OES_texture_float')
+      );
+    }
+
+    if (colorBufferHalfFloat) {
+      supported =
+        supported &&
+        Boolean(
+          gl.getExtension('EXT_color_buffer_float') ||
```

Comment:
I think it could be worth some explanation around the fact that these are overlaping WebGL1 and WebGL2 extensions. It is not our mess but I had to read up on all the extensions to review this code.

Reply:
Sure. This just simplified things a bit since I have to check several for the float test, and in general, it seems redundant to me to split up the extensions by API version. You can check the extension, and if it's written against a different API version, it just won't be available.

Reply:
Ok the comment lines helps, thanks :+1: